Sources/SwiftSyntaxBuilder/generated/Tokens.swift

```diff
@@ -18,543 +18,543 @@ import SwiftSyntax
 /// Namespace for commonly used tokens with default trivia.
 public extension TokenSyntax{
   
-/// The `associatedtype` keyword
-static var `associatedtype`: TokenSyntax{
+    /// The `associatedtype` keyword
```

Comment:
Still something that's not working 100% correctly here

Reply:
Yes, this needs to be fixed before we can merge it. Not sure what’s going on here either.